stumpy/floss.py

```diff
@@ -482,7 +483,9 @@ def update(self, t):
         Segmentation (FLOSS).
         """
         self._T[:] = np.roll(self._T, -1)
+        self._T_isfinite[:] = np.roll(self._T_isfinite, -1)
```

Comment:
I'm a little bit confused here. I see that you are indeed using `self._T_isfinite` to track whether individual elements in T are finite. However, above, it is clear that it contains subsequences rather than elements. I'm surprised that the final output is "right"